library/std/src/net/tcp.rs

```diff
@@ -548,6 +548,46 @@ impl TcpStream {
         self.0.ttl()
     }
 
+    // Sets the value for the `IPV6_UNICAST_HOPS` option on this socket.
+    ///
+    /// This value sets the unicast hop limit field that is used in every packet
+    /// sent from this socket.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// #![feature(ipv6_hop_limit)]
+    /// use std::net::TcpStream;
+    ///
+    /// let stream = TcpStream::connect("[::1]:12345")
+    ///                        .expect("Couldn't connect to the server...");
+    /// stream.set_hop_limit_v6(88).expect("set_hop_limit_v6 call failed");
+    /// ```
+    #[unstable(feature = "ipv6_hop_limit", issue = "47727")]
```

Comment:
Please file a new dedicated tracking issue.

Reply:
Done in https://github.com/rust-lang/rust/issues/139166.
The issue number has been updated accordingly in the code.